pkgs/applications/kde/ktouch.nix

```diff
@@ -18,9 +18,10 @@
     buildInputs = [
       kconfig kconfigwidgets kcoreaddons kdeclarative ki18n
       kitemviews kcmutils kio knewstuff ktexteditor kwidgetsaddons
-      kwindowsystem kxmlgui qtscript qtdeclarative kqtquickcharts
+      kwindowsystem kxmlgui qtscript qtdeclarative kqtquickcharts qtx11extras 
+      xorg.libxkbfile
     ]
-    ++ lib.optionals withX11 (with xorg; [ libxcb libxkbfile qtx11extras ]);
-  #propagatedUserEnvPkgs = [ kipi-plugins libkipi (lib.getBin kinit) ];
-  enableParallelBuilding = true;
```

Comment:
did you unset `enableParallelBuilding` on purpose?

Reply:
Oops, no. Fix incoming (waiting for dependencies to build)

Reply:
resolved now